IFATQCZZ-64 - Added Factory pattern to initialize WebDriver

kostyantins opened the pull request:
IFATQCZZ-13 - Added TestCase "As a Customer I want to see "Ordering" and "User Info" tab after login"
IFATQCZZ-64 - Added Factory pattern to initialize WebDriver

Commit:
IFATQCZZ-64 - Added Factory pattern to initialize WebDriver
IFATQCZZ-13 - Added TestCase "As a Customer I want to see "Ordering" and "User Info" tab after login"

Commit:
IFATQCZZ-64 - Added Factory pattern to initialize WebDriver
IFATQCZZ-13 - Added TestCase "As a Customer I want to see "Ordering" and "User Info" tab after login"
With correct

toleksyn commented on src/test/java/academy/softserve/edu/utils/WebDriverFactory.java:
```diff
@@ -0,0 +1,43 @@
+
+        switch (browser) {
+
+            case FIREFOX:
```
merge this with default:
case FIREFOX:
default:
.....

toleksyn commented on src/test/java/academy/softserve/edu/tests/TestCaseSwitchTabsByCustomer.java:
```diff
@@ -0,0 +1,113 @@
+    }
+
+    @Test
+    public void defaultUserInfoPageTest() {
```
all tests methods should start with "test"

Commit:
IFATQCZZ-64 - Added Factory pattern to initialize WebDriver
IFATQCZZ-13 - Added TestCase "As a Customer I want to see "Ordering" and "User Info" tab after login"
With Test methods correct

toleksyn commented on src/test/java/academy/softserve/edu/tests/TestCaseSwitchTabsByCustomer.java:
```diff
@@ -0,0 +1,113 @@
+    }
+
+    @Test
+    public void TestDefaultUserInfoPage() {
```
code convention for method names!

Commit:
IFATQCZZ-64 - Added Factory pattern to initialize WebDriver
IFATQCZZ-13 - Added TestCase "As a Customer I want to see "Ordering" and "User Info" tab after login"
With Test methods correct

Commit:
IFATQCZZ-64 - Added Factory pattern to initialize WebDriver
IFATQCZZ-13 - Added TestCase "As a Customer I want to see "Ordering" and "User Info" tab after login"
With Test methods correct 02

Commit:
IFATQCZZ-64 - Added Factory pattern to initialize WebDriver
IFATQCZZ-13 - Added TestCase "As a Customer I want to see "Ordering" and "User Info" tab after login"
With Test methods correct and code convention
